identity/signature.go

```diff
@@ -19,10 +23,14 @@ func (signature *Signature) Hex() string {
 	return hex.EncodeToString(signature.raw)
 }
 
-func (signature *Signature) String() string {
-	return string(signature.raw)
+func (signature *Signature) Base64() string {
+	return base64.StdEncoding.EncodeToString(signature.Bytes())
 }
 
 func (signature *Signature) Bytes() []byte {
 	return signature.raw
 }
+
+func (signature Signature) EqualsTo(other Signature) bool {
+	return bytes.Equal(signature.raw, other.raw)
```

Comment:
Is it safe to ignore cases when `signature` or `other` is `nil`?

Reply:
Since receiver and other are values but not pointers they cannot be nil. If you try to pass nil here you will get compiler error